.npmrc

```diff
@@ -1,2 +1,6 @@
+lockfileVersion=2
 scripts-prepend-node-path=true
 legacy-peer-deps=true
+engine-strict=false
+ignore-engines=true
+loglevel=error
```

Comment:
looks not nice

Reply:
use from here:
https://github.com/Tinkoff/taiga-ui/blob/main/.npmrc#L3